flask_resources/loaders.py

```diff
@@ -39,12 +39,20 @@ def wrapper(self, *args, **kwargs):
         """
         # Checking Content-Type is the responsibility of the loader
         loaders = self.resource.config.request_loaders
+        default_content_type = getattr(
+            self.resource.config, "default_content_type", None
+        )
+
+        request_content_type = request.content_type
+
+        if not request_content_type and default_content_type:
+            request_content_type = default_content_type
```

Comment:
minor: the logic is a bit messy to follow, what about the following (simpler IMHO)

```suggestion
        request_content_type = request.content_type
        
        if not request_content_type:
            request_content_type =getattr(self.resource.config, "default_content_type", None)
```